app/assets/javascripts/welcome.js

```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```

Comment:
I think we can also delete these empty files (welcome.js, welcome.scss, welcome_helper.rb)